embeddings/pipeline/preprocessing_pipeline.py

```diff
@@ -52,120 +58,134 @@ def run(self) -> TransformationResult:
         return result
 
 
-class FlairTextClassificationPreprocessingPipeline(
-    PreprocessingPipeline[str, datasets.DatasetDict, Corpus]
+# Mypy currently properly don't handle dataclasses with abstract methods  https://github.com/python/mypy/issues/5374
```

Comment:
Consider adding todo and followup task

Reply:
Good idea, but it's outside of scope of this PR, as we have similar cases in other places.

Reply:
Added to #120